vendor/engines/secure_rooms/config/locales/en.yml

```diff
@@ -15,6 +15,9 @@ en:
       selection_needed: "Must select Account"
       rules_failed: "No rule generated a verdict"
 
+  pages:
+    admin_occupancies: Occupancies
```

Comment:
I'm leaning towards "Rooms", but it's something we can always change later after client feedback.